<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPatientDialog</class>
 <widget class="QDialog" name="EditPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="SurnameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Иванов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Виталий</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PatronymicLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Юрьевич</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="BirthdayDateEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2001</year>
     <month>5</month>
     <day>14</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddressLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Кострома</string>
   </property>
  </widget>
  <widget class="QComboBox" name="GenderComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="InsurancePolicyLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>157-852-956-321</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChoosePhotoButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Photo...</string>
   </property>
  </widget>
  <widget class="QLabel" name="FilePathLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>841</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
